### ISSUE TITLE
packages for Ubuntu 24 seems to be missing on packages.microsoft.com

### ISSUE THREAD

**Issue by wfurt** [microsoft/msquic#4367]
### Describe the bug

Install fresh Ubuntu 24 and add packages.microsoft.com as package source. 

### Affected OS

- [ ] Windows
- [X] Linux
- [ ] macOS
- [ ] Other (specify below)

### Additional OS information

Ubuntu 24

### MsQuic version

v2.3

### Steps taken to reproduce bug

run pack age manages and try to install libmsquic. 

### Expected behavior

install should succeed 

### Actual outcome

install fails

**Comment by ShreyasZare** [comment]
I hope this issue is fixed soon since new deployments are facing issues due to missing packages.

**Comment by ShreyasZare** [comment]
Is there any ETA available for this issue? If its going to take too long then users can be advised accordingly to downgrade to Ubuntu 22.04 instead of waiting.

**Comment by ShreyasZare** [comment]
I can see that `libmsquic` version `2.4.0` is available now for Ubuntu 24.04 on [Microsoft packages](https://packages.microsoft.com/ubuntu/24.04/prod/pool/main/libm/libmsquic/). But, it does not work with .NET applications.

Just tested this with Ubuntu Server 24.04 (fully updated) and I see following error with .NET app:
```
System.PlatformNotSupportedException: System.Net.Quic is not supported on this platform: Unable to load MsQuic library version '2'.
   at System.Net.Quic.QuicConnection.ConnectAsync(QuicClientConnectionOptions options, CancellationToken cancellationToken)
```

The `QuicConnection.IsSupported` property also returns `false`.

**Comment by nibanks** [comment]
Closing as this should be fixed now.
